sdk/jobs/pipelines/1h_automl_in_pipeline/automl-image-classification-multiclass-in-pipeline.ipynb

```diff
@@ -0,0 +1,420 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# 2. Basic pipeline job with Image Classification Multiclass task\n",
+    "\n",
+    "## 2.1 Build pipeline"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# note that the used docker image doesn't suit for all size of gpu compute. Please use the following command to create gpu compute if experiment failed\n",
+    "# !az ml compute create -n gpu-cluster --type amlcompute --min-instances 0 --max-instances 4 --size Standard_NC12"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Define pipeline\n",
+    "@pipeline(\n",
+    "    description=\"AutoML Image Clasiification Multiclass Pipeline\",\n",
+    "    default_compute=\"gpu-cluster\",\n",
+    ")\n",
+    "def automl_image_classification(\n",
+    "    image_classification_train_data,\n",
+    "    image_classification_validation_data\n",
+    "):\n",
+    "    # define the automl image_classification task with automl function\n",
+    "    image_classification_node = image_classification(\n",
+    "        training_data=image_classification_train_data,\n",
+    "        validation_data=image_classification_validation_data,\n",
+    "        target_column_name=\"label\",\n",
+    "        primary_metric=\"Accuracy\",\n",
+    "        properties={\"_automl_internal_label\": \"latest\",},\n",
+    "        # currently need to specify outputs \"mlflow_model\" explictly to reference it in following nodes \n",
+    "        outputs={\"best_model\": Output(type=\"mlflow_model\")},\n",
+    "    )\n",
+    "    \n",
+    "    image_classification_node.set_limits(timeout_minutes=180)\n",
+    "    \n",
+    "    image_classification_node.extend_search_space(\n",
+    "        [\n",
+    "            ImageClassificationSearchSpace(\n",
+    "                model_name=Choice([\"vitb16r224\", \"vits16r224\"]),\n",
+    "                learning_rate=Uniform(0.001, 0.01),\n",
+    "            ),\n",
+    "            ImageClassificationSearchSpace(\n",
+    "                model_name=Choice([\"seresnext\", \"resnet50\"]),\n",
+    "                learning_rate=Uniform(0.001, 0.01),\n",
+    "            ),\n",
+    "        ]\n",
+    "    )\n",
+    "    image_classification_node.set_sweep(\n",
+    "        max_trials=20,\n",
+    "        max_concurrent_trials=4,\n",
+    "        sampling_algorithm=\"Random\",\n",
+    "        early_termination=BanditPolicy(evaluation_interval=2, slack_factor=0.2, delay_evaluation=6),\n",
+    "    )\n",
+    "\n",
+    "    command_func = command(\n",
+    "        inputs=dict(\n",
+    "            automl_output=Input(type=\"mlflow_model\")\n",
+    "        ),\n",
+    "        command=\"ls ${{inputs.automl_output}}\",\n",
+    "        environment=\"AzureML-sklearn-0.24-ubuntu18.04-py37-cpu:1\"\n",
```

Comment:
Why is this specific image used ? Does this command run inference using the mlflow model ?